.github/workflows/reproducibility.yaml

```diff
@@ -10,6 +10,9 @@ jobs:
   build_reproducibility_index:
     runs-on: ubuntu-20.04
 
+    permissions:
+      contents: write
```

Comment:
Does this really work? Publishing reproducibility index on a PR after merge is failing with `status: 403`. I wonder if it is happening because of the changes related to the `codecov` issue.

Reply:
Good catch. This is failing because the steps also adds a comment to the PR by posting an issue comment (https://docs.github.com/en/rest/reference/issues#create-an-issue-comment). This requires it to also have the issues:write permission.